Makefile

```diff
@@ -1,20 +1,24 @@
 # spaceteam Makefile
 
 CC = g++
-CFLAGS = -std=c++0x -g -Wall
+CFLAGS = -std=c++11 -Wall -Werror
 GTEST_DIR = googletest/googletest
 TEST_CFLAGS = -std=c++0x -isystem $(GTEST_DIR)/include -pthread
 
 TARGET = config_parser
 TEST_TARGET = config_parser_test
 
 PARSER_SOURCE = config_parser.cc config_parser_main.cc
+SERVER_SOURCE = connection.cc server.cc main.cc config_parser.cc
 TEST_SOURCE = config_parser_test.cc config_parser.cc $(GTEST_DIR)/src/gtest_main.cc
 
-all: parser
+all: parser server
 
 parser:	$(PARSER_SOURCE)
-	$(CC) $(CFLAGS) -o $(TARGET) $(PARSER_SOURCE)
+	$(CC) $(CFLAGS) -g -o $(TARGET) $(PARSER_SOURCE)
+
+server: $(SERVER_SOURCE)
```

Comment:
Is the executable supposed to be called `webserver`? See the [assignment 2 spec](https://docs.google.com/document/d/1UG9n7rnITq3XwIc2ijGXXJk0Y2OTwvU6KXz7mTs-yOM/pub)